python/src/main.py

```diff
@@ -0,0 +1,6 @@
+from prices import create_app
+
+
+if __name__ == "__main__":
+    app = create_app()
```

Comment:
@martinsson that would not work because it returns a tuple:

![tuple](https://user-images.githubusercontent.com/1094774/225905629-95e9401d-0677-42a6-b4ba-29d2d19b15ff.png)

If we go with this approach, I'd suggest to extract the app part like this:

```suggestion
    app, connection = create_app()
```

![after](https://user-images.githubusercontent.com/1094774/225905721-b3ce75bd-b7df-4fba-83c1-6990f50dee1d.png)